booking.py

```diff
@@ -107,6 +108,106 @@ def add_tagged_friends(friends):
 
     return Dataset(dataset_name, ntuples)
 
+def dataset_from_crownoutput(
+        dataset_name,
+        file_names,
+        folder,
+        files_base_directory,
+        friends_base_directories):
+    """Create a Dataset object from a list containing the names
+    of the ROOT files (e.g. [root_file1, root_file2, (...)]):
+        ntuple1: /file_base_dir/root_file1/folder/ntuple
+            friend1: /friend1_base_dir/root_file1/folder/ntuple
+            friend2: /friend2_base_dir/root_file1/folder/ntuple
+        ntuple2: /file_base_dir/root_file2/folder/ntuple
+            friend1: /friend1_base_dir/root_file2/folder/ntuple
+            friend2: /friend2_base_dir/root_file2/folder/ntuple
+        ntuple3: /file_base_dir/root_file3/folder/ntuple
+            friend1: /friend1_base_dir/root_file3/folder/ntuple
+            friend2: /friend2_base_dir/root_file3/folder/ntuple
+        (...)
+
+    Args:
+        dataset_name (str): Name of the dataset
+        file_names (list): List containing the names of the .root
+            files
+        folder (str): Name of the TDirectoryFile in each .root file
+        files_base_directory (str): Path to the files base directory (directories)
+        friends_base_directories (str, list): List of paths to
+            the friends base directory (directories)
+
+    Returns:
+        dataset (Dataset): Dataset object containing TTrees
+    """
+    def get_quantities_per_variation(path_to_root_file):
+        root_file = TFile(path_to_root_file)
+        if root_file.IsZombie():
+            logger.fatal('File {} does not exist, abort'.format(path_to_root_file))
+            raise FileNotFoundError
+        quantities_per_vars={}
+        quantities_with_variations = root_file.Get("ntuple").GetListOfLeaves() 
+        for qwv in quantities_with_variations:
+            qwv_name = qwv.GetName()
+            if "__" in qwv_name:
+                quantity,var = qwv_name.split("__")
+                if var not in quantities_per_vars.keys():
+                    quantities_per_vars[var] = []
+                quantities_per_vars[var].append(quantity)
+        root_file.Close()
+        return quantities_per_vars
+
+
+    def get_full_tree_name(folder, path_to_root_file, tree_name):
```

Comment:
You do not need this function any more, do you?

Reply:
Or do you use it to verify that all root files exist? Then it might still be useful but can probably be replaced with a faster check since we do not need to make sure the folder exists since everything is called `ntuple` anyway.